packages/ocamlfind/ocamlfind.1.6.2/opam

```diff
@@ -10,11 +10,15 @@ build: [
   [make "all"]
   [make "opt"] { ocaml-native }
 ]
-install: [make "install"]
+install: [
+  [make "install"]
+  ["cp" "ocaml" bin] {preinstalled}
```

Comment:
Just checking, what happens with `make "install"` in a preinstalled build ?  (I guess nothing). This is just because when you build ocaml it builds `ocaml` at the root of the distrib so the `file/ocaml` would be overwritten. Maybe, it's better to call the script `ocaml-stub` and do a `"cp" "ocaml-stub" "%{bin}%/ocaml"`